src/main/flight/dyn_notch_filter.c

```diff
@@ -282,7 +282,7 @@ static FAST_CODE_NOINLINE void dynNotchProcess(void)
             }
 
             // Search for N biggest peaks in frequency spectrum
-            for (int bin = (sdftStartBin + 1); bin < sdftEndBin; bin++) {
+            for (int bin = sdftStartBin; bin <= sdftEndBin; bin++) {
```

Comment:
This can introduce a critical **array index out of bounds** error and can cause the FC to lock up with unfortunate dyn notch settings.